packages/bruno-app/src/utils/collections/index.js

```diff
@@ -417,7 +417,9 @@ export const transformRequestToSaveToFilesystem = (item) => {
   });
 
   if (itemToSave.request.body.mode === 'json') {
-    itemToSave.request.body.json = replaceTabsWithSpaces(itemToSave.request.body.json);
+    const copyItemToSave = JSON.parse(itemToSave);
```

Comment:
As this bug originates from the fact that object coming from redux store are [not extensible](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Cant_define_property_object_not_extensible), cloning the items sent to `saveMultipleRequests` action you could avoid this error if a similar mutation is done elsewhere.
You can see the same approach in `saveRequest` action:
```js
const collectionCopy = cloneDeep(collection);
const item = findItemInCollection(collectionCopy, itemUid);
```

Reply:
II have changed this